tests/testthat/test-utils.R

```diff
@@ -51,3 +51,18 @@ test_that("padding_lst applies padding to a flextable based on indentation level
   padded_ft <- padding_lst(sample_ft, sample_indents)
   expect_is(padded_ft, "flextable")
 })
+
+block_text <- "Line 1\nLine 2\nLine 3\nLine 4\nLine 5\nLine 6"
+
+test_that("split_text_block - splits text block into equal parts", {
+  n <- 2
+  result <- split_text_block(block_text, n)
+  expected_result <- list("Line 1\nLine 2", "Line 3\nLine 4", "Line 5\nLine 6")
+  expect_equal(result, expected_result)
+})
```

Comment:
I think a better test would be
```suggestion
test_that("split_text_block - splits text block into blocks no longer than n lines", {
  l <- 5
  block_text <- paste(paste(rep("Line", l), 1:l), collapse = "\n")
  n <- 2
  result <- split_text_block(block_text, n)
  result_lines <- lapply(result, function(x) strsplit(x, "\n")[[1]])
  lapply(result_lines, function(x) expect_lte(length(x), n))
})
```